linen_examples/pixelcnn/pixelcnn.py

```diff
@@ -34,17 +34,18 @@
 
 published at ICLR '17 (https://openreview.net/forum?id=BJrFC6ceg).
 """
+
+# pytype: disable=wrong-arg-count
```

Comment:
I'm not against module level disables but I think a comment to explain why it's needed is nice so we can work towards removing it.

Reply:
I created #620 and linked it everywhere where I disabled pytype errors. Hopefully we can resolve the issue soon and then remove all the module level disable statements.